### PR TITLE
Add ORACLE related env variables to DB image

### PR DESCRIPTION
**Pull request by akinfermo**
Change-Id: Ie907942096aa02c730ee45672ae214c730ae0f0a

### CODE REVIEW

File: oracle/build/dbimage/image_build.sh [hunk 1]
```diff
@@ -239,17 +241,25 @@ execute_command() {
...
+    ORACLE_HOME="/u01/app/oracle/product/${DB_VERSION}/db"
+    ORACLE_BASE="/u01/app/oracle"
+  fi
+
   if [ -z "${TAG}" ]; then
     TAG="gcr.io/${GCR_PROJECT_ID}/oracle-database-images/oracle-${DB_VERSION}-${EDITION}-${IMAGE_NAME_SUFFIX}:latest"
   fi
 
   if [ "${LOCAL_BUILD}" == true ]; then
-    BUILD_CMD=$(echo sudo docker build --no-cache --build-arg=DB_VERSION=${DB_VERSION} --build-arg=CREATE_CDB=${CREATE_CDB} --build-arg=CDB_NAME=${CDB_NAME} --build-arg=CHARACTER_SET=${CHARACTER_SET} --build-arg=MEM_PCT=${MEM_PCT} --build-arg=EDITION=${EDITION} --build-arg=PATCH_VERSION=${PATCH_VERSION} --tag=$TAG .)
```
Comment (yian-zhou) [inline oracle/build/dbimage/image_build.sh:247]: FYI, removing sudo would require users set up sudoless docker. See https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user

**Review by yian-zhou — APPROVED** [review]
LGTM. Thanks Ash!

File: oracle/build/dbimage/image_build.sh (hunk 1 above)
Comment (akinfermo) [reply oracle/build/dbimage/image_build.sh:247]: Indeed!

Comment (akinfermo) [reply oracle/build/dbimage/image_build.sh:247]: In my experience, most people set up sudo-less docker.

**Pull request merged**